rust/connlib/tunnel/src/tests/transition.rs

```diff
@@ -265,6 +291,32 @@ where
         })
 }
 
+fn port_from_resource(
```

Comment:
The idea here is to more often pick ports within the allowed range, otherwise allowed packets w/o internet resource(which in the future shouldn't even happen)  would be too sparse

Reply:
Should we add some coverage logs to the gateway perhaps?

Reply:
Added some coverage for blocked packets [here](https://github.com/firezone/firezone/pull/7148/files#diff-33b1012b6eff6423dfac798d70927fb02fab4fdccab5a337c88bcb0823bf0639R117) I'll add something for passing packets.